SPECS/openssh/openssh.spec

```diff
@@ -182,22 +169,22 @@ useradd test -G root -m
 sudo -u test -s /bin/bash -c "PATH=$PATH TEST_SSH_UNSAFE_PERMISSIONS=1 make tests"
 
 %pre server
-getent group sshd >/dev/null || groupadd -g 50 sshd
-getent passwd sshd >/dev/null || useradd -c 'sshd PrivSep' -d %{_sharedstatedir}/sshd -g sshd -s /bin/false -u 50 sshd
+getent group sshd >/dev/null || groupadd --system sshd
+getent passwd sshd >/dev/null || useradd --system --comment 'Privilege Separated SSH' --home-dir %{_sharedstatedir}/sshd --gid sshd --shell /bin/false sshd
 
 %preun server
-%systemd_preun sshd.service sshd-keygen.service
+%{systemd_preun} sshd.service sshd-keygen.service
```

Comment:
Can you remove the braces around the `systemd` macros that the linter added? Sorry for the bother, I have a fix for that in the pipeline, I've been meaning to publish a new linter version but other work has gotten in the way.

Reply:
Done